app/states/unified_sqlmodels.py

```
@@ -0,0 +1,1467 @@
+                name=entity_name,
+                address=address,
+                person=person
+            )
+            if entity:
+                person.entity = entity
+            
+            return person
+        
+        return None
+    
+    def build_address(self, raw_data: Dict[str, Any], entity_role: str) -> Optional[UnifiedAddress]:
```

Ruff (0.14.7)

Unused method argument: `entity_role`

(ARG002)